decidim-comments/spec/cells/decidim/comments/comment_form_spec.rb

```diff
@@ -33,8 +33,8 @@ module Decidim::Comments
         expect(subject).to have_css(".hashtags__container textarea#add-comment-DummyResource-#{commentable.id}[maxlength='1000']")
         expect(subject).to have_css("#add-comment-DummyResource-#{commentable.id}-remaining-characters")
         expect(subject).to have_css("input.alignment-input[name='comment[alignment]'][value='0']", visible: :hidden)
-        expect(subject).to have_css("input[name='comment[commentable_gid]']", visible: :hidden)
-        expect(subject).to have_css("button", text: "Send")
+        expect(subject).to have_css("input[name='comment[commentable_gid]']", visible: :hidden) # rubocop:disable Capybara/SpecificMatcher
```

Comment:
We can fix this with:
```suggestion
        expect(subject).to have_field(name: "comment[commentable_gid]", type: :hidden)
```

Reply:
Actually we can't... I have tried and failed. 
I have found this [StackOverflow](https://stackoverflow.com/questions/48870804/why-is-the-have-field-capybara-matcher-returning-no-matches-for-a-hidden-fie) question, that led me to this [Capybara issue](https://github.com/teamcapybara/capybara/issues/1017) . 

> Well, pretty much. We don't go out of our way to support stuff which isn't specific to testing stuff which is visible to users. If I'd have my way, things would be more restrictive than they are, but pragmatism gets in the way, as is the case with :visible => false.
>
> As far as checking for hidden fields, as you noticed, it's possible, it just isn't easy, and that's kind of where we draw the line. When someone uses have_field to assert the value of a field in the context of an integration test, then they don't expect that to find hidden fields, and that is the use case we optimize for.
>
> We provide you the power of low level APIs so you can work around these defaults, we even provide you the power to build abstractions using the add_selector API, we just don't optimize for that use case.

Reply:
I tested it with this expectation and the test passed, though.

I checked with git blame and this spec is originating from my comments refactor (#6498). But I don't remember why it was added in the first place.

Reply:
You are are right. I was trying with "visible" and that one is indeed failing.